README-template.md

```diff
@@ -426,11 +435,165 @@ Please note there are a few caveats:
 * Reading from views is **disabled** by default. In order to enable it, set the `viewsEnabled` configuration
   property to `true`.
 
+## Reading from BigQuery table snapshots
+
+The connector supports reading from [BigQuery table snapshots](https://cloud.google.com/bigquery/docs/table-snapshots-intro).
+
+A BigQuery table snapshot preserves the contents of a table (called the base table) at a particular
+time. You can save a snapshot of a current table, or create a snapshot of a table as it was at any
+time in the past seven days.
+
+To link a Hive table to a BigQuery table snapshot, simply specify the snapshot's name in the
+`bq.table` table property, for example:
+
+```sql
+CREATE TABLE mytable (abc BIGINT, xyz STRING)
+STORED BY 'com.google.cloud.hive.bigquery.connector.BigQueryStorageHandler'
+TBLPROPERTIES (
+    'bq.table'='myproject.mydataset.mysnapshot'
+);
+```
+
+## Read consistency
+
+As stated in the [BigQuery Storage API documentation](https://cloud.google.com/bigquery/docs/reference/storage),
+read sessions are based on a snapshot isolation model (Note: this is unrelated to
+[BigQuery table snapshots](https://cloud.google.com/bigquery/docs/table-snapshots-intro)). All
+consumers read based on a specific point in time. The default snapshot time is based on the read
+session creation time (i.e. when the `SELECT` query is initiated), but consumers may read data from an
+earlier snapshot.
```

Comment:
Do we allow this through Hive connector? If so, please add an example.

Reply:
Ok, so time travel isn't supported by the connector yet. I've clarified the documentation.
The feature should be easy to add some customers explicitly ask for it.